src/pages/unexpected-error/home/home.tsx

```diff
@@ -0,0 +1,122 @@
+import React, { useState, useEffect } from 'react';
+
+import globalStyles from 'assets/stylesheets/global-styles.module.scss';
+import { AppLink } from 'routes/app-link';
+import { RouteName } from 'routes/routes';
+import { Button } from 'common/button';
+import { Avatar } from 'common/avatar';
+import { Modal } from 'common/modal';
+import { ModalSizes } from 'common/modal/modal';
+import { TextField, TextFieldStatus } from 'common/text-field';
+import { TextArea } from 'common/text-area';
+import { mailFormat } from 'helpers/utils';
+import styles from './home.module.scss';
+import { ReactComponent as MailSVG } from '../../assets/icons/mail.svg';
+import { ReactComponent as CloseSVG } from '../../assets/icons/close.svg';
+
+const Home = () => {
+  const [openModal, setOpenModal] = useState(false);
+  const closeModal: () => void = () => {
```

Comment:
What about put this function closeModal() at the bottom of the useState() lines to respect an order in the code?

Reply:
@agustinasalmanton This is from the template, I didn't created it